audit/result.go

```diff
@@ -142,6 +142,8 @@ func (r *Result) ForContainer(cnr cid.ID) {
 // Zero Result has nil key. Return value MUST NOT be mutated: to do this,
 // first make a copy.
 //
+// The returning value is not mutation-free, changing it leads to data corruption. Make a copy if you need to change it.
```

Comment:
`The value returned shares memory with the structure itself, so changing it can lead to....`

It's not entirely correct since this memory is not a part of the structure, but it should be sufficient.

Reply:
Updated